client/core/trade.go

```diff
@@ -950,7 +953,8 @@ func (t *trackedTrade) isRedeemable(ctx context.Context, match *matchTracker) bo
 			return ready
 		}
 		// If we're the taker, check the confirmations anyway so we can notify.
-		confs, _, err := t.wallets.fromWallet.Confirmations(ctx, match.MetaData.Proof.TakerSwap)
+		confs, err := t.wallets.fromWallet.SwapConfirmations(ctx, match.MetaData.Proof.TakerSwap,
+			match.MetaData.Proof.Script, match.MetaData.Stamp)
```

Comment:
Although the tests in trade_simnet_test.go pass, I now see the error below logged like

`[ERR] CORE:1: error getting confirmation for our own swap transaction: swap has been spent`

While the logic here or in `isSwappable` does not change the outcome, we now see this log and a `MatchNote` will have 0 for confirmations while previously there was the actual confirmations and no error.